src/main/kotlin/com/ampnet/mailservice/service/impl/UserMailServiceImpl.kt

```diff
@@ -197,4 +202,19 @@ class UserMailServiceImpl(
     private fun getUser(userUuid: String): UserResponse =
         userService.getUsers(listOf(userUuid)).firstOrNull()
             ?: throw ResourceNotFoundException("Missing user: $userUuid")
+
+    private fun hasProjectWhichCanReceiveInvestment(coop: String): Boolean {
+        val projects = projectService.getActiveProjects(coop)
+            .associateBy { UUID.fromString(it.uuid) }
+        val wallets = walletService.getWalletsByOwner(projects.keys.toList())
+        wallets.forEach { wallet ->
+            val projectBalance = blockchainService.getBalance(wallet.hash)
+            val expectedFunding = projects[UUID.fromString(wallet.owner)]?.expectedFunding
+            if (projectBalance.isLessThan(expectedFunding)) return true
```

Comment:
```suggestion
            val projectBalance = blockchainService.getBalance(wallet.hash) ?: 0
            val expectedFunding = projects[UUID.fromString(wallet.owner)]?.expectedFunding ?: 0
            if (expectedFunding > projectBalance) return true
```

Reply:
This change would return true in case blockchain service returns null in which case we are not sure if the balance is greater or less than expected funding. I would leave the implementation as is.

Reply:
Ok, then why not simplify with:
```kotlin
            val projectBalance = blockchainService.getBalance(wallet.hash) ?: return false
```